app/serializers/serialize_exercise_instructions.rb

```diff
@@ -75,22 +71,24 @@ def tasks
       end
   end
 
+  def instructions
+    [
+      Markdown::RenderMarkdown.(exercise.git.instructions).strip,
+      Markdown::RenderMarkdown.(exercise.git.instructions_append).strip
+    ].join("\n").strip
```

Comment:
```suggestion
    ].join("\n\n").strip
```

I'm not sure if this is correct, but we need to decide whether we're adding a full line break or not. I feel like it's safer that we do, in which case we need the double newline I think.